### ISSUE TITLE
The repository is not mounted

### ISSUE THREAD

**Issue by NelDav** [equalitie/ouisync-app#441]
OS: Windows (64-bit)

---
When I initially add a repository to Ouisync, the repository is shown in the "Ouisync" drive. But as soon I reboot the computer, the repositories are still shown in the Ouisync application but no longer displayed in the filesystem.
Also, when I try to access the files, I get an error:

> The repository is not mounted

Reinstalling the application does not fix the problem. The only way to get access to those repositories again is to remove the files located in `AppData\Roaming\org.equalitie\ouisync\repositories` and add the repositories once more. Then, the repositories are mounted again, and I can access the files until I reboot my computer.

---
log:
Coming soon

**Comment by IvanaBlzvc** [comment]
Hi @NelDav - we are trying to reproduce the error you get.  Can you please tell us what version of Ouisync you are using?  And what version of Windows? Many thanks.

**Comment by NelDav** [comment]
Hi @IvanaBlzvc, thanks for your reply. I am using `Windows 11, 23H2` and Ouisync 0.8.2.

**Comment by J-Pabon** [comment]
> Hi @IvanaBlzvc, thanks for your reply. I am using `Windows 11, 23H2` and Ouisync 0.8.2. I also attached the log now. Just to clarify, I think the errors in line 3 and 4 are because I manually removed the `.ouisyncdb` file, since deleting the repository via the Ouisync application, does not work in case the repository is not mounted.

What about Dokan?
Which version are you using?

Did you install it using the Ouisync app option the first time it launched?

**Comment by NelDav** [comment]
> Did you install it using the Ouisync app option the first time it launched?

Yes, I did install it during the first launch. I am using `Dokan 2.1.0.1000 (x64)`, which isn't  the most recent dokan version but the one that was downloaded automatically. The log tells:

> 2024-11-04T10:18:04.615447Z DEBUG The Dokan version installed is supported: sameVersion :: app{_MainPageState} :: ffi\src\lib.rs:300

**Comment by NelDav** [comment]
I guess, dokan might indeed be the problem. I uninstalled the old version and installed the latest one. After the installation, the repositories were mounted again. But then after a reboot, they are not mounted anymore.

**Comment by IvanaBlzvc** [comment]
https://redmine.equalit.ie/issues/31245
